rust/schema.py

```diff
@@ -119,15 +118,31 @@ class Function(Declaration):
     body: Expr | child
 
 
-#     Missing,
+@rust.doc_test_signature("() -> ()")
```

Comment:
hmm, maybe this should be the default, and there should be a pragma to not wrap the doc code in a function (for top-level declarations)?

Reply:
That would be handy indeed.